src/boggle/core.clj

```diff
@@ -1,8 +1,8 @@
 (ns boggle.core
-    (:use [clojure.math.numeric-tower :only [abs]]
-          [clojure.java.io :as io])
-    (:require [clojure.string :as string])
-    (:gen-class))
+  (:require [clojure.string :as string]
+            [clojure.java.io :as io]
+            [clojure.math.numeric-tower :refer [abs]])
```

Comment:
May I ask what the reasoning is behind this change?

Reply:
Mostly based on [this section of the style guide](https://github.com/bbatsov/clojure-style-guide#prefer-require-over-use).  I think the motivation for that is because `:use` makes it easier to overwrite or shadow var names since it automatically refers them into the current namespace.